base/applications/network/tracert/tracert.cpp

```diff
@@ -557,6 +569,52 @@ RunTraceRoute()
     return Success;
 }
 
+static bool
+GetUlongOptionInRange(
+    _In_ int argc, 
+    _In_ wchar_t *argv[], 
+    _Inout_ int *i, 
+    _Out_ ULONG *Value,
+    _In_  ULONG MinimumValue,
+    _In_  ULONG MaximumValue)
+{
+    ULONG ParsedValue = 0;
+    
+    // check input arguments
+    if (argv == NULL || i == NULL || Value == NULL)
+        return false;
+    
```

Comment:
These cannot be null because the way the function is annotated (all these parameters are NOT optional) forbids this. (If the caller of the function doesn't respect what the function requires, the function will crash and it's their problem.)
```suggestion
```